ztp/source-crs/validatorCRs/informDuValidator.yaml

```diff
@@ -4,6 +4,11 @@ metadata:
   name: $mcp
   annotations:
     ran.openshift.io/ztp-deploy-wave: "10000"
+    # first time TALM observes this policy as compliant, it won't move on.
+    # In future reconcilations if the specified number of seconds has passed 
+    # since the first compliancy time and policy is still compliant, then TALM
+    # will move on from this policy
+    soakSeconds: "30"
```

Comment:
Can we make this annotation more TALM specific? If, for example, the Policy team implements a similar annotation we don't want to have a name collision. eg ran.openshift.io/soakSeconds

Reply:
Is this the right place for the annotation though? @serngawy Policy gen copies it from this MCP CR template to the policy?

Reply:
this annotation `soakSecond` will be in the MachineConfigPool that exist in the policy not in the policy annotation. Is that what you are looking to do ?

Reply:
In the end, it needs to be in policy annotation for TALM to recognize. However, from a modelling perspective, it does make some sense to attach it to the actual CR template at the source. How about policy gen checking this value in CRs and use the greatest at the policy level? Sort of similar to the checks on the waves.